packages/file_selector/file_selector_windows/example/lib/open_image_page.dart

```diff
@@ -48,8 +48,8 @@ class OpenImagePage extends StatelessWidget {
           children: <Widget>[
             ElevatedButton(
               style: ElevatedButton.styleFrom(
-                backgroundColors: Colors.blue,
-                foregroundColors: Colors.white,
+                backgroundColor: Colors.blue,
+                foregroundColor: Colors.white,
```

Comment:
Does this mean you did the migration by hand? If so, I would definitely encourage using `flutter fix` in the future when possible, as it's both easier, and avoids potential errors like this (or more subtle ones, like accidentally swapping colors during the migration).

Reply:
Yes I did it by hand. I didn't know about the `flutter fix` command lol.
Thanks for the suggestion!